crates/next-core/js/src/dev/hmr-client.ts

```diff
@@ -116,6 +116,15 @@ function aggregateUpdates(
   const key = resourceKey(msg.resource);
   const aggregated = chunksWithUpdates.get(key);
 
+  if (msg.type === "issues" && aggregated == null && hasIssues) {
```

Comment:
Nit: Looks like this can be merged with the if statements below?

Reply:
could be, but imo makes it harder to read